pkg/bindings/testing/testing.go

```diff
@@ -0,0 +1,370 @@
+package testing
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/containers/podman/v5/pkg/bindings"
+	"github.com/containers/podman/v5/pkg/domain/entities/types"
+)
+
+func CreateStorageLayer(ctx context.Context, options *CreateStorageLayerOptions) (*types.CreateStorageLayerReport, error) {
```

Comment:
this is technically part of a public API as we support pkg/bindings for consumers which is very bad

Reply:
Moved it to an internal package.